src/main/java/org/candlepin/async/tasks/CloudAccountOrgSetupJob.java

```diff
@@ -71,6 +73,7 @@ public CloudAccountOrgSetupJob(CloudRegistrationAdapter cloudAdapter, OwnerCurat
      *     if an exception occurs during job execution
      */
     @Override
+    @WithSpan
```

Comment:
does the '@WithSpan' annotation generate a traceID by itself? Or does it only propagate an existing tradeID if one exists?
I thought its just the former, in which case there will be no traceID to propagate to adapter and then later on to RHSM-API

Reply:
it should, because every span has ID(Span and trace). [Traces](https://opentelemetry.io/docs/concepts/signals/traces/). But i will verify that after i will create test enviroment for that. But then depends if they want it as seperate trace or include in the trace where they are calling it